src/assets/i18n/en.json5

```diff
@@ -5639,4 +5639,332 @@
+  "ldn-enable-service.notification.success.content": "The service status has been updated",
+  "ldn-service-delete.notification.success.title": "Successful Deletion",
+  "ldn-service-delete.notification.success.content": "The service has been deleted",
+  "ldn-service-delete.notification.error.title": "Failed Deletion",
+  "ldn-service-delete.notification.error.content": "The service has not been deleted",
+  "service.overview.reset-form.reset-return": "Cancel",
+  "service.overview.delete": "Delete service",
+  "ldn-edit-service.title": "Edit service",
+  "ldn-edit-service.form.label.name": "Name",
+  "ldn-edit-service.form.label.description": "Description",
+  "ldn-edit-service.form.label.url": "Service URL",
+  "ldn-edit-service.form.label.ldnUrl": "LDN Inbox URL",
+  "ldn-edit-service.form.label.inboundPattern": "Inbound Pattern",
+  "ldn-edit-service.form.label.noInboundPatternSelected": "No Inbound Pattern",
+  "ldn-edit-service.form.label.selectedItemFilter": "Selected Item Filter",
+  "ldn-edit-service.form.label.selectItemFilter": "No Item Filter",
+  "ldn-edit-service.form.label.automatic": "Automatic",
+  "ldn-edit-service.form.label.addInboundPattern": "+ Add more",
+  "ldn-edit-service.form.label.submit": "Save",
+  "ldn-edit-service.breadcrumbs": "Edit Service",
+  "ldn-service.control-constaint-select-none": "Select none",
+
+  "ldn-register-new-service.notification.error.title": "Error",
+  "ldn-register-new-service.notification.error.content": "An error occurred while creating this process",
+  "ldn-register-new-service.notification.success.title": "Success",
+  "ldn-register-new-service.notification.success.content": "The process was successfully created",
+
+  "info.coar-notify-support.title": "Notify Support",
+
+  "info.coar-notify.breadcrumbs": "Notify Support",
+
+  "submission.sections.notify.info": "The selected service is compatible with the item according to its current status. {{ service.name }}: {{ service.description }}",
+
+  "item.qa-event-notification.check.notification-info": "There are {{num}} pending review to check",
+
+  "item.qa-event-notification-info.check.button": "Check",
+
+  "mydspace.qa-event-notification.check.notification-info": "There are {{num}} pending review to check",
+
+  "mydspace.qa-event-notification-info.check.button": "Check",
+
+  "item.page.endorsed-by": "Endorsement",
+
+  "item.page.is-reviewed-by": "Review",
+
+  "item.page.is-supplemented-by": "Dataset",
+
+  "item.page.endorsment": "Endorsment",
+
+  "item.page.review": "Review",
+
+  "item.page.dataset": "Dataset",
+  "menu.section.icon.ldn_services": "LDN Services overview",
+  "menu.section.services": "LDN Services",
+
+  "menu.section.services_new": "LDN Service",
+
+  "mydspace.import": "Import",
+
+  "mydspace.notification.suggestion": "We found <b>{{count}} publications</b><br> in the {{source}} that seems to be related to your profile.<br> Please <a href='/suggestions/{{suggestionId}}'>review the suggestions</a>",
```

Comment:
These look like the text that we fixed in the OpenAIRE Publication Claim PR?  Are you sure this PR is updated with all the latest code from Publication Claim?  This text was fixed and it was changed to no longer be an "href"

Reply:
mydspace. prefixed labels changed in the suggestions PR were leftovers from Coar main branch